composer.json

```diff
@@ -19,78 +19,132 @@
+        "symfony/asset": "4.4.*",
+        "symfony/console": "4.4.*",
+        "symfony/dotenv": "4.4.*",
+        "symfony/expression-language": "4.4.*",
+        "symfony/flex": "^1.3.1",
+        "symfony/form": "4.4.*",
+        "symfony/framework-bundle": "4.4.*",
+        "symfony/http-client": "4.4.*",
+        "symfony/intl": "4.4.*",
+        "symfony/mailer": "4.4.*",
+        "symfony/monolog-bundle": "^3.1",
+        "symfony/process": "4.4.*",
+        "symfony/property-access": "4.4.*",
+        "symfony/property-info": "4.4.*",
+        "symfony/security-bundle": "4.4.*",
+        "symfony/serializer": "4.4.*",
+        "symfony/translation": "4.4.*",
+        "symfony/twig-bundle": "^4.4",
+        "symfony/validator": "4.4.*",
+        "symfony/web-link": "4.4.*",
+        "symfony/yaml": "4.4.*",
+        "twig/extensions": "^1.0",
+        "twig/extra-bundle": "^2.12|^3.0",
+        "twig/twig": "^2.12|^3.0",
         "zendframework/zend-feed": "^2.0",
         "zendframework/zend-servicemanager": "^2.0",
         "zendframework/zend-uri": "^2.0",
-        "pagerfanta/pagerfanta": "^2.4",
-        "knplabs/knp-menu-bundle": "^2.1",
-        "ezyang/htmlpurifier": "^4.6",
-        "nelmio/cors-bundle": "^1.4",
-        "cebe/markdown": "^1.1",
-        "algolia/algoliasearch-client-php": "^2.0",
-        "seld/signal-handler": "^1.1",
-        "php-http/httplug-bundle": "^1.11",
-        "php-http/guzzle6-adapter": "^1.1",
-        "zendframework/zenddiagnostics": "^1.4",
-        "graze/dog-statsd": "^0.4.2",
-        "incenteev/composer-parameter-handler": "^2.1",
-        "beelab/recaptcha2-bundle": "^2.3",
-        "babdev/pagerfanta-bundle": "^2.7"
+        "zendframework/zenddiagnostics": "^1.4"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^4.2",
-        "phpunit/phpunit": "^8.0"
+        "symfony/browser-kit": "^4.4",
+        "symfony/css-selector": "^4.4",
+        "symfony/debug-bundle": "^4.4",
+        "symfony/maker-bundle": "^1.0",
+        "symfony/phpunit-bridge": "^5.1",
+        "symfony/stopwatch": "^4.4",
+        "symfony/var-dumper": "^4.4",
+        "symfony/web-profiler-bundle": "^4.4"
+    },
+    "config": {
+        "preferred-install": {
+            "*": "dist"
+        },
+        "sort-packages": true,
+        "platform": {
+            "php": "7.4.10"
+        }
+    },
+    "autoload": {
+        "psr-4": {
+            "App\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "App\\Tests\\": "tests/"
+        }
+    },
+    "replace": {
+        "paragonie/random_compat": "2.*",
+        "symfony/polyfill-ctype": "*",
+        "symfony/polyfill-iconv": "*",
+        "symfony/polyfill-php71": "*",
+        "symfony/polyfill-php70": "*",
+        "symfony/polyfill-php56": "*"
     },
     "scripts": {
-        "symfony-scripts": [
-            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
```

Comment:
is it intended to remove that script while you kept the `incenteev/composer-parameter-handler` dependency ?